### PR TITLE
Pylint

### PR DESCRIPTION
**Pull request by z4y4ts**
TODO:
- [x] Fix all pylint errors
- [x] Make Travis green

### CODE REVIEW

File: video_xblock/video_xblock.py [hunk 1]
```diff
@@ -290,7 +289,7 @@ def validate_field_data(self, validation, data):
 
         if data.href == '':
             return
-        for player_name, player_class in BaseVideoPlayer.load_classes():
+        for player_name, player_class in BaseVideoPlayer.load_classes():  # pylint: disable=unused-variable
```
Comment (z4y4ts) [inline video_xblock/video_xblock.py]: Don't disable it here.
Instead `player_name` -> `_`

File: video_xblock/video_xblock.py [hunk 2]
```diff
@@ -23,8 +22,8 @@
 from pycaption import detect_format, WebVTTWriter
 from webob import Response
 
-from backends.base import BaseVideoPlayer, html_parser
-from settings import ALL_LANGUAGES
+from backends.base import BaseVideoPlayer, html_parser  # pylint: disable=relative-import
```
Comment (z4y4ts) [inline video_xblock/video_xblock.py]: Did you try `from video_xblock.backends.base import ...` instead?

File: video_xblock/backends/base.py [hunk 3]
```diff
@@ -1,3 +1,7 @@
+"""
+Base Video player plugin
```
Comment (z4y4ts) [inline video_xblock/backends/base.py]:
```python
"""
Backend classes are responsible for video platform specific logic such as
validation, interaction with the platform via API and player rendering to end user.

Base Video player plugin.
"""
```

File: video_xblock/video_xblock.py (hunk 2 above)
Comment (sendr) [inline video_xblock/video_xblock.py]: yes, ImportError: No module named backends.base

**Pull request merged**